src/Store.js

```diff
@@ -51,6 +51,26 @@ class Store {
     }
   }
 
+  /**
+   * builds an instance of a model but does not add it to the store
+   * ```
+   * kpiHash = { name: "A good thing to measure" }
+   * kpi = store.build('kpis', kpiHash)
+   * kpi.name
+   * => "A good thing to measure"
+   * ```
+   * @method add
```

Comment:
^^ forgot to update the docs here
```suggestion
   * @method build
```